src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1881,7 +1890,7 @@ public override BoundNode VisitCall(BoundCall node)
 
             //if (this.State.Reachable) // PROTOTYPE(NullableReferenceTypes): Consider reachability?
             {
-                _resultType = method.ReturnType;
+                _resultType = GetAdjustedType(method.ReturnType);
```

Comment:
>GetAdjustedType [](start = 30, length = 15)

I'm not sure about this. Just to confirm, here's a scenario:

```C#
T Copy<T>(T t) ...
void M<U>(U u)
{
    if (u == null) throw;
    var x = Copy(u);
    x.ToString(); // do we expect a warning here?
}
```

Reply:
I would not expect a warning here: we've done the null check on `u`, so it should be `U!`. It would be a different question if the T was constrained to be `class?`, but with unconstrained I think this should be fine.

---
In reply to: [207688966](https://github.com/dotnet/roslyn/pull/28956#discussion_r207688966) [](ancestors = 207688966)